README.md

```diff
@@ -1,2 +1,10 @@
 # InteractiveProgramming
 This is the base repo for the interactive programming project for Software Design, Spring 2016 at Olin College.
+
+Our Project Proposal:
+https://docs.google.com/document/d/10Eax7S3gga7mB4fEm4NYQ3jXghqH5RktQyJAh4wDo6k/edit?usp=sharing
+
+Final Code can be found in snakeV2.py.
+
+Our Final Reflection:
+https://docs.google.com/document/d/1e_sk4rJcuHJ4s9-rXBqwoRf4FXzqjexgqnUPTLMEz0o/edit?usp=sharing
```

Comment:
Sensible reflection. Scheduling and planning ahead is a huge skill, but it gets better with practice 😄 As well, realizing that OpenCV was out of scope was reasonable. It requires a lot of work, even before implementing a whole game for it.